WalletWasabi.Gui/CommandLine/Daemon.cs

```diff
@@ -84,9 +86,13 @@ internal async Task RunAsync(string walletName, bool mixAll, bool keepMixAlive)
 					return;
 				}
 
-				await TryQueueCoinsToMixAsync(mixAll, password);
+				KeyManager destinationKeyManager = TryGetKeyManagerFromWalletName(destinationWalletName);
+				bool isDestinationSpecified = keyManager.ExtPubKey != destinationKeyManager.ExtPubKey;
```

Comment:
Why we didn't simply compare `walletName != destinationWalletName` here?

Reply:
To handle the case when the name differs but the extpub is the same?